django_extensions/templates/django_extensions/graph_models/label.dot

```diff
@@ -18,11 +18,13 @@
     {{ model.label }}{% if model.abstracts %}<BR/>&lt;<FONT FACE="Helvetica Italic">{{ model.abstracts|join:"," }}</FONT>&gt;{% endif %}
     </FONT></TD></TR>
   {% if not disable_fields %}{% for field in model.fields %}
+  {% if not disable_abstract_fields or not field.abstract %}
```

Comment:
I really dislike the `double-not-or if` construct. It's a shame Django templates does not support `if not (disable_abstract_fields and field.abstract)`.

Besides it's ugliness, I even prefer:
```
{% if disable_abstract_fields and field.abstract %}
{% else %}
...
{% endif %}
```

To it, just because `double-not-or if` is so hard to read.

Reply:
I was very close to adding it to the list of unsure points yesterday :100: 
I had in mind the alternative you are suggesting, I will make the change alongside the tests.

Reply:
yeah it's a shame there is not really a better alternative as far as I know.. Either it's the `{% else %}` variant or creating a template tag or injecting it into the template context from code. 

Would love for Django templates to support expressions like `{% if not (a and b) %}` :-)

Reply:
I would need a little help regarding testing. It seems that doing : 
```python
    out = StringIO()
    call_command('graph_models',
                 app_label=['django_extensions'],
                 include_models=['AbstractInheritanceTestModelChild'],
                 disable_abstract_fields=False,
                 stdout=out)
    output = out.getvalue()
```
Doesn't capture the prints from `graph_models.py` (i.e. `output=''`) which is annoying for testing things on those prints.
Should I change the prints by something else ? Or maybe working with files ?

Reply:
It should work now.

Added a commit with fixed for this in `graph_models` command and also added some very basic tests to make sure graph_model executes. You can add your tests to `test_graph_models.py` if you want.

Reply:
Thanks for the help ! 
I have made the modification in the template and added the tests.

Reply:
Thanks for the help ! I have made the modification in the template and added the tests.